### ISSUE TITLE
Hosting 

### ISSUE THREAD

**Issue by hemanta212** [OSAC/OSAC-Forum#8]
Since Heroku is no more free, I guess pythonanywhere is the way to go. Are there any other good free solutions? 

feel free to mention if you need any help here.

**Comment by Shashwat-pd** [comment]
Even Azure gives some free credits for basic projects. Not so sure about the details though.

**Comment by hemanta212** [comment]
yeah azure has such facility if you get a github student pack. Should last at least 8 month and is renewable yearly. Did you get yours?

**Comment by hemanta212** [comment]
Update: Tested it using a free hosting called render 

https://forum.osac.org.np

Turns out any service providing custom domain routing can be used to attach to our osac website domain.

A long term solution i would recommend to you is to use your Azure credits to host the django app and route to our website domain.

Let me see if you need any help on this, apply for azure student accounts to get credits and research a bit on this one.

**Comment by Aju100** [comment]
@hemanta212 @Shashwat-pd 
Try out these too
- https://render.com/
- https://fly.io/
- https://railway.app/

**Comment by hemanta212** [comment]
Thank you for the pointers dai.

render and fly io seem unfeasible for this project as the former deletes the db after 90 days and latter requires credit card for the free tier.

railway.app seems promising but the limit is 5 dollars worth of credits, however running this project with postgres will bill 8$ acc to their calculator. 
But i think railway.app is still doable/worth a try.

**Comment by Aju100** [comment]
Great

**Comment by ajutamangdev** [comment]
any updates on this @hemanta212 bhai ?
